test_scripts/Defects/6_1/1917_Media_navigation_and_nonmedia_apps_must_be_activated_during_active_embedded_audio_source.lua

```diff
@@ -0,0 +1,156 @@
+-- Steps to reproduce:
+-- 1) User activates non-media app_3
+--    HMI -> SDL: OnAppDeactivated ( <appID_1> )
+--    HMI -> SDL: SDL.ActivateApp ( <appID_3 )
+-- Expected result:
+-- 1) SDL -> media app_1 : OnHMIStatus (LIMITED, AUDIBLE)
+-- 2) SDL -> HMI: SDL.ActivateApp (SUCCESS)
+-- 3) SDL -> non-media app_3 : OnHMIStatus (FULL, NOT_AUDIBLE)
+-- Note: navigation app_2 still at LIMITED and AUDIBLE
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+local hmi_values = require('user_modules/hmi_values')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local mediaAppId = 1
+local naviAppId = 2
+local nonMediaAppId = 3
+
+--[[ Applications Configuration ]]
+common.getConfigAppParams(mediaAppId).appHMIType = { "MEDIA" }
+common.getConfigAppParams(mediaAppId).isMediaApplication = true
+common.getConfigAppParams(naviAppId).appHMIType = { "NAVIGATION" }
+common.getConfigAppParams(naviAppId).isMediaApplication = false
+common.getConfigAppParams(nonMediaAppId).appHMIType = { "DEFAULT" }
+common.getConfigAppParams(nonMediaAppId).isMediaApplication = false
+
+--[[ Local Functions ]]
+local function getHMIValues()
+  local params = hmi_values.getDefaultHMITable()
+  params.BasicCommunication.MixingAudioSupported.attenuatedSupported = true
+  return params
+end
+
+local function activateApp(pAppId)
+  if not pAppId then pAppId = 1 end
+  local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(pAppId) })
+  common.getHMIConnection():ExpectResponse(requestId)
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus")
+  :ValidIf(function(_, data)
+    if config["application"..pAppId].registerAppInterfaceParams.isMediaApplication == false and
+      config["application"..pAppId].registerAppInterfaceParams.appHMIType[1] == "DEFAULT"
+      then
+        return data.payload.audioStreamingState == "NOT_AUDIBLE" and
+          data.payload.hmiLevel == "FULL" and
+          data.payload.systemContext == "MAIN"
+      else
+        return data.payload.audioStreamingState == "AUDIBLE" and
+          data.payload.hmiLevel == "FULL" and
+          data.payload.systemContext == "MAIN"
+    end
+  end)
+end
+
+local function deactivateApp(pAppId)
+  if not pAppId then pAppId = 1 end
+  common.getHMIConnection():SendNotification("BasicCommunication.OnAppDeactivated",
+    { appID = common.getHMIAppId(pAppId) })
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus")
+  :ValidIf(function(_, data)
+    if config["application"..pAppId].registerAppInterfaceParams.isMediaApplication == true or
+      config["application"..pAppId].registerAppInterfaceParams.appHMIType[1] == "NAVIGATION"
```

Comment:
```suggestion
      config["application"..pAppId].registerAppInterfaceParams.appHMIType[1] == "NAVIGATION"
```
`NAVIGATION` app is `STREAMABLE` from `LIMITED`